src/NUnitFramework/framework/Constraints/ConstraintExpression.cs

```diff
@@ -949,11 +949,27 @@ public SamePathOrUnderConstraint SamePathOrUnder(string expected)
         /// Returns a constraint that tests whether the actual value falls 
         /// within a specified range.
         /// </summary>
+        /// <remarks>from must be less than or equal to true</remarks> 
```

Comment:
What about `The <paramref name="from"/> value must be less than or equal to the <paramref name="to"/> value.`?